src/fortran_routing/Reservoir_Binding/kernels/fortran/Reservoirs/test_binding.c

```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
```

Comment:
Note: this may not be needed in the final commit...

Reply:
Nels suggested to me to keep this testing file in order for there to be a binding test within the Reservoir directory.